### ISSUE TITLE
borderColor is not defined in LabelProps

### ISSUE THREAD

**Issue by katestud** [primer/components#822]
The [Label component](https://primer.style/components/Label) supports the `borderColor` prop, but it looks like the typescript definition of `Label` doesn't implement it:

```
<Label variant="small" borderColor="green.5">
    My Label
</Label>
```

This throws the warning:
```
  Property 'borderColor' does not exist on type 'IntrinsicAttributes & LabelProps & { children?: ReactNode; }'
```

Using `sx=` does work, though!

```
<Label variant="small" sx={{ borderColor: "green.5" }}>
    My Label
</Label>
```

Thanks! ❤️ 🌹 🙇‍♀️

**Comment by emplums** [comment]
Thanks for reporting this @katestud! :) I have shipped a fix in 19.1.0 :)

**Comment by katestud** [comment]
Amazing, thank you, @emplums !
